lib/requests/request_executor.rb

```diff
@@ -169,7 +169,7 @@ def execute_command(command, server_node)
       
       begin
         response = http_client(server_node).request(request)
-      rescue OpenSSL::SSL::SSLError => ssl_exception
+      rescue OpenSSL::SSL::SSLError
         request_exception = unauthorized_error(server_node, request)
```

Comment:
@a-serdukov-mobilunity We'd still like to know what's the `ssl_exception` message and it's not passed anywhere.